sdk/automanage/Azure.ResourceManager.Automanage/src/Generated/ConfigurationProfileCollection.cs

```diff
@@ -72,7 +72,7 @@ internal static void ValidateResourceId(ResourceIdentifier id)
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentException"> <paramref name="configurationProfileName"/> is an empty string, and was expected to be non-empty. </exception>
         /// <exception cref="ArgumentNullException"> <paramref name="configurationProfileName"/> or <paramref name="data"/> is null. </exception>
-        public virtual async Task<ArmOperation<ConfigurationProfileResource>> CreateOrUpdateAsync(WaitUntil waitUntil, string configurationProfileName, ConfigurationProfileData data, CancellationToken cancellationToken = default)
+        public virtual async Task<ArmOperation<ConfigurationProfileResource>> CreateOrUpdateAsync(WaitUntil waitUntil, string configurationProfileName, AutomanageConfigurationProfileData data, CancellationToken cancellationToken = default)
```

Comment:
I suggest we also prepend Automanage on this collection to align with others.
And this name might be quite generic and other RPs may have similar things like this

Reply:
I missed this change, thanks!